### PR TITLE
Create Word Count extension

### PR DESCRIPTION
**Pull request by itsmingjie**
## Description

The Word Count extension counts the number of characters, words, sentences, and paragraphs in some text. 

- The command will automatically count any text in your clipboard. To clear, use `⌘ + E`.
- Whitespace is ignored by default when counting characters. To toggle whitespace counting, use `⌘ + W`.

Requested by this Slack thread: https://raycastcommunity.slack.com/archives/C01AC2X0GMN/p1658322740595279

## Screencast

CleanShot - CleanShot - 2022-07-22 at 19 15 35

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by itsmingjie** [comment]
Seems like the failing CI is due to the same issue as this thread on Slack: https://raycastcommunity.slack.com/archives/C028DTXTW3Y/p1658455463991239. Will try to snap one on a Retina screen.

File: extensions/word-count/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "access": "public",
+  "author": "itsmingjie",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "count",
+      "title": "Count",
```
Comment (danielsequeira) [inline extensions/word-count/package.json:16]:
```suggestion
      "title": "Count",
      "subtitle": "Word Count",
```

File: extensions/word-count/src/count.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  useEffect(() => {
+    (async () => {
+      const clipboard = await Clipboard.readText();
+
+      if (clipboard) {
+        setText(clipboard.trim());
+        showToast({
+          style: Toast.Style.Success,
+          title: `Loaded text from clipboard`,
```
Comment (danielsequeira) [inline extensions/word-count/src/count.tsx:18]:
```suggestion
          title: `Text loaded from clipboard`,
```

**Comment by danielsequeira** [comment]
Hi @itsmingjie,

Thanks for your contribution.

I just added a few small comments, nothing major 🙂

**Review by danielsequeira — APPROVED** [review]

**Comment by tjex** [comment]
The plugin does not count characters correctly.    
image   
image   
image

**Comment by danielsequeira** [comment]
@tjex please report it as a bug in https://www.raycast.com/itsmingjie/word-count

**Comment by tjex** [comment]
Ahh sorry. DIdn't see this way. Thanks

**Pull request merged**